homeassistant/components/upnp/__init__.py

```diff
@@ -46,43 +40,13 @@
 
 PLATFORMS = [Platform.BINARY_SENSOR, Platform.SENSOR]
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We should keep `CONFIG_SCHEMA` around to prevent invalid configuration.

```py
CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
```

Reply:
done